.github/workflows/ci.yml

```diff
@@ -45,6 +44,24 @@ jobs:
            fi
     
     build:
+        runs-on: "ubuntu-latest"
```

Comment:
We agreed on removing build, right? As the build happens when the tests are run.

Reply:
sorry, merge conflict mistake. removed